### ISSUE TITLE
Toolbar Drag Nullpointer

### ISSUE THREAD

**Issue by GregHib** [weisJ/darklaf#56]
```
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
	at com.github.weisj.darklaf.ui.toolbar.DarkToolBarUI.dragTo(DarkToolBarUI.java:201)
	at com.github.weisj.darklaf.ui.toolbar.DarkToolBarUI.lambda$new$0(DarkToolBarUI.java:46)
	at java.desktop/javax.swing.Timer.fireActionPerformed(Timer.java:317)
	at java.desktop/javax.swing.Timer$DoPostEvent.run(Timer.java:249)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```
Occurs on Windows when dragging a floating JToolbar between monitors of different resolutions.

**Comment by weisJ** [comment]
Does this exclusively happen when the resolution is different and/or the monitor is different?

**Comment by weisJ** [comment]
Looks like that `MouseInfo.getPointerInfo()` can return `null` when moving between monitors as the pointer information is temporarily unavailable. Fixed.

**Comment by weisJ** [comment]
Fix available with [`1.4.0.0`](https://github.com/weisJ/darklaf/releases/tag/v1.4.0.0)
